charts/bandstand-cron-job/templates/pv.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if and .Values.s3Volumes (gt (len .Values.s3Volumes) 0) }}
+{{- range $volume := .Values.s3Volumes }}
+{{- $relName := printf "%s-%s" .Release.Name $volume.name }}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ printf "%s-pv" $relName }}
+  labels:
+    {{- include "bandstand-cron-job.labels" $ | nindent 4 }}
+spec:
+  capacity:
+    storage: {{ $volume.persistent | default "5Gi" }}
+  volumeMode: Filesystem
+  accessModes:
+    - ReadOnlyMany
+  persistentVolumeReclaimPolicy: Retain
+  csi:
+    driver: s3.csi.aws.com
+    volumeHandle: {{ $volume.volumeHandle | default (print "s3-" $relName "-volume") }}
```

Comment:
What is this $volume.volumeHandle attribute?

Reply:
I think you miss my point. I'm wondering why you are making it settable by the user at all - $volume comes from the values file